lib/mix/tasks/firmware.burn.ex

```diff
@@ -63,9 +63,14 @@ defmodule Mix.Tasks.Firmware.Burn do
     # Create a temporary .fw file that fwup.exe is able to access
```

Comment:
The "create a temporary .fw file" comment should be moved down to line 69.5.

Reply:
ah yes, i'll move that down